creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -8,6 +9,25 @@ const _ = require('lodash')
 // We add a namespace prefix to differentiate internal metrics from those exported by different exporters from the same host
 const NamespacePrefix = 'audius_cn_'
 
+const MetricLabels = Object.freeze({
+  ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM: {
```

Comment:
it's just a string here so this works but I think maybe you're saying I should make it so we access this like `MetricLabels[MetricNames.ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM]` instead of `MetricLabels.ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM`?

Reply:
yep exactly, wdyt

Reply:
I think that would make types easier instead of using strings. just pushed changes for it

Reply:
coo